s5-state_and_event/src/components/Student Class/index.js

```diff
@@ -0,0 +1,35 @@
+import './studentclass_style.css';
+import React from 'react';
+
+class StudentClass extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {name: "Illuminati", age: 18};
+        this.increaseAge = this.increaseAge.bind(this);
+        this.resetInfo = this.resetInfo.bind(this)
+    };
+
+    increaseAge() {
+        this.setState((prevState) => {
+            return {name: "Illuminati", age: prevState.age+1};
```

Comment:
mình thay đổi tuổi thì chỉ cần set lại age thôi, không cần phải set lại cái name nữa

em đọc doc trên react ở đây
https://reactjs.org/docs/state-and-lifecycle.html#state-updates-are-merged

Reply:
Vâng, cái này cũng là do em nghịch thêm để hiểu rõ hơn thôi ạ!